src/components/Layout/Navbar/NavbarDesktop.tsx

```diff
@@ -3,68 +3,60 @@ import { Dispatch, SetStateAction } from "react";
 import Image from "next/image";
 import { HOME_URL, NAV_LINKS } from "./NavbarConstant";
 import Link from "next/link";
-import { useSelector } from "react-redux";
-import { RootState } from "@/src/store";
-import UserProfile from "./components/UserProfile";
-import GithubLogin from "./components/GithubLogin";
 
-import styles from './navbar.module.css'
+import { UserProfileWithGitHubLogin } from "./components/UserProfileWithGitHubLogin";
 
-export default function NavbarDesktop({ isDropdownVisible, setIsDropdownVisible } : {
+import styles from "./navbar.module.css";
+
+export default function NavbarDesktop({
+  isLoggedIn,
+  isDropdownVisible,
+  firstName,
+  imageURL,
+  setIsDropdownVisible,
+}: {
+  isLoggedIn: boolean;
   isDropdownVisible: boolean;
+  firstName: string | null;
+  imageURL: string | null;
   setIsDropdownVisible: Dispatch<SetStateAction<boolean>>;
 }) {
-  const { isLoggedIn } = useSelector((state: RootState) => state.global);
-  let profileComponent;
-
-  if (isLoggedIn) {
-    profileComponent = <UserProfile isDropdownVisible={isDropdownVisible} setIsDropdownVisible={setIsDropdownVisible} />;
-  } else {
-    profileComponent = <GithubLogin />;
-  }
-
   const navItems = NAV_LINKS.map((link) => (
-    <ListItem
-      key={link.id}
-      flex={"flex"}
-      alignItems={"center"}
-      margin={"2px"}
-    >
+    <ListItem key={link.id} flex={"flex"} alignItems={"center"} margin={"2px"}>
       <Link href={link.link}>
-        <Text
-          className={styles.navlinks}
-        >
-          {link.name}
-        </Text>
+        <Text className={styles.navlinks}>{link.name}</Text>
       </Link>
     </ListItem>
   ));
 
   return (
-   <nav className={styles.navbar} data-testId="navbarDesktop">
-     <UnorderedList
-      listStyleType="none"
-      className={styles.navbar_menu}
-    >
-      <ListItem>
-        <Link href={HOME_URL}>
-          <Image
-            src="/images/Real-Dev-Squad@1x.svg"
-            width={45}
-            height={45}
-            alt="RDS logo"
+    <nav className={styles.navbar} data-testId="navbarDesktop">
+      <UnorderedList listStyleType="none" className={styles.navbar_menu}>
```

Comment:
Can the list style be shifted to the class?

Reply:
can we keep it as it is cause it renders the `UserProfileWithGithubLogin` component 
and the style is one line only